google_drive/google_calendar.py

```diff
@@ -18,34 +18,42 @@
 # Constants ###########################################################
 DEFAULT_CALENDAR_ID = "edx.org_lom804qe3ttspplj1bgeu1l3ak@group.calendar.google.com"
 DEFAULT_CALENDAR_URL = (
-    'https://www.google.com/calendar/embed?mode=Month&src={}&showCalendars=0'.format(DEFAULT_CALENDAR_ID))
+    'https://www.google.com/calendar/embed?mode=Month&src={}&showCalendars=0&hl=en-us'.format(DEFAULT_CALENDAR_ID))
 CALENDAR_TEMPLATE = "/templates/html/google_calendar.html"
 CALENDAR_EDIT_TEMPLATE = "/templates/html/google_calendar_edit.html"
 
 
+def _(text):
+    """
+    Dummy ugettext.
```

Comment:
Well, because XBlocks are initialized as soon as possible and that makes Django freaks out with an exception!

All the other XBlocks uses this trick to overcome this limitation. Honestly, this a pretty useless trick, because XBlocks don't yet support i18n for field name, so :man_shrugging: 

But everyone's doing this in the hope that we can do something about it soon, I'm also optimistic about that as well so I do the same.

Reply:
Hmm.  Would `ugettext_lazy` work? https://docs.djangoproject.com/en/1.11/topics/i18n/translation/#lazy-translation  If that breaks something in the depths of the XBlock machinery, I'm fine with this as is.